src/sidebar/components/Annotation/Annotation.js

```diff
@@ -111,7 +116,7 @@ function Annotation({
                 Saving...
               </div>
             )}
-            {annotation && showActions && (
+            {annotation && showActions && isSaved(annotation) && (
```

Comment:
Given the current UI design and workflow, `showActions` implicitly implies that the annotation is saved, but code is now more explicit that the action bar won't be shown for unsaved annotations.

Reply:
Yes, good, I like.